README.md

```diff
@@ -1216,6 +1216,23 @@ a_n_u_p_p_e_r_c_a_s_e_d_w_o_r_d
 -->
 ```
 
+###camelize
+
+Converts a string to camelCase<br/>
+usage: ```string | camelize: [upperFirst:boolean|optional]```<br/>
```

Comment:
The camelCase filter with `upperFirst` enabled is basically a PascalCase filter. Maybe it's better to split them up into two filters.